src/s2b_tent_city_chase.tw2

```diff
@@ -0,0 +1,530 @@
+
+“Time to step out of the shadows, sister.”
+
+I froze. I recognized the voice — [[but it wasn't Leo's|Tent City - Three Rings recruiter]].]
+{
+  (set: $badChoices to 0)
+  (set: $recruiterTriedExcuse to false)
+  (set: $recruiterTalkedShit to false)
+}
+
+::Tent City - Three Rings recruiter
+No, it was that god-fearing ninja who hustled for the so-called “Temple of Three Rings.” He was standing along the side of the path, not four feet away from me. I could see the fistful of church flyers clutched in his left hand. Fuck.
+
+“That’s right sister, I seen you. An’ I also seen the look in your eyes. You think hiding is gonna keep you safe.”
+
+He took a step closer.
+
+(click: "He took a step closer")[“But every soul gots a ticket to the Carnival: non-transferable, non-refundable, non-fuck-with-able."
+
+He paused.
+
+"Only question now is, which show will your soul see?”
+
+(if: $recruiterTriedExcuse is false)[>> [[Make an excuse|Tent City - Three Rings recruiter - make excuse]]]
+>> [[Quote verse|Tent City - Three Rings recruiter - quote verse]]
+(if: $recruiterTalkedShit is false)[>> [[Talk shit|Tent City - Three Rings recruiter - talk shit]]]
+>> [[Take tract|Tent City - Three Rings recruiter - take tract]]]
+
+::Tent City - Three Rings recruiter - make excuse
+“Uhh, I don’t wanna waste your time, man," I stammered. "My mom keeps me on a pretty short leash. She doesn’t like the look of y’all to begin with, and I’m out past curfew already.”
+
+He leaned in even closer, and I caught the scent of gin-spiked Moon Mist wafting on his breath. “Judge not, sister. That goes double for momma. If she gots worries, why, a night in the Three Rings is just the thing.”
+
+I sighed. Clearly he wasn't buying my bullshit.
+
+>> [[Quote verse|Tent City - Three Rings recruiter - quote verse]]\
+(if: $recruiterTalkedShit is false)[
+>> [[Talk shit|Tent City - Three Rings recruiter - talk shit]]]
+>> [[Take tract|Tent City - Three Rings recruiter - take tract]]
+{
+  (set: $badChoices to it +1)
+  (set: $recruiterTriedExcuse to true)
+}
+
+::Tent City - Three Rings recruiter - quote verse
+“'Ring three, the ICP, look if you want but I wouldn't lay a hand on me,'” I sang, jerking my head toward his. “'That's how you get fucked up, I’ll squeeze your windpipe shut.'”
+
+Preacher-man cackled, and he rasped back: “'You can call me a weirdo, call me a freak, call me Don Knotts cuz I'm gettin' knotted every week!' Hah, sister, you know your shit.” He kept laughing as he patted me on the back, waving me on my way. [[I was only too happy to oblige|Tent City - $nextScene]].
+{
+  (set: $tentChasePartTwo to (random: 1))
+  (if: $tentChasePartTwo is 0)[(set: $nextScene to "Rip's Rydas")]
+  (else:)[(set: $nextScene to "Ornery Kids")]
+}
```

Comment:
The hooks to the next passage on the Three Rings recruiter section don't work. But, I was able to fix it pretty quickly by moving the code at the end of the passage to the beginning. I'd suggest doing that here and in the "take tract" passage.

Otherwise, the code works great and the next passage selection seems random! I tested several times to make sure.